src/services/users/users.validations.ts

```diff
@@ -1,32 +1,44 @@
-// add users validations
-// validate if email is exist
-// remove user - should be blocked! (hooks - disallow('extyernal)
 import Joi from 'joi';
-
+const firstNameSchema = Joi.string().trim().min(1).max(100);
+const lastNameSchema = Joi.string().trim().min(1).max(100);
+const emailSchema = Joi.string().trim().email();
+const passwordSchema = Joi.string();
+const titleSchema = Joi.string().trim().min(1).max(100);
 
 export const createUserSchema = Joi.object({
-  firstName: Joi
-    .string()
-    .trim()
-    .min(1)
-    .max(100)
-    .required(),
-  lastName: Joi
-    .string()
-    .trim()
-    .min(1)
-    .max(100)
-    .required(),
-  email: Joi
-    .string()
-    .trim()
-    .email()
-    .required(),
-  password: Joi
-    .string()
-    .required()
+  firstName: firstNameSchema.required(),
+  lastName: lastNameSchema.required(),
+  email: emailSchema.required(),
+  password: passwordSchema.required(),
+  title: titleSchema
+});
+
+export const patchUserSchema = Joi.object({
+  firstName: firstNameSchema,
+  lastName: lastNameSchema,
+  email: emailSchema,
+  password: passwordSchema,
+  title: titleSchema,
+});
+
+export const updateUserSchema = Joi.object({
```

Comment:
`export const replaceUserSchema = createUserSchema`

Reply:
fixed e63ee2f79bcab0301e1774bea05285cfad975b6d

Reply:
I also meant to reuse `createUserSchema` b/c they are equal